lib/gateway/Shard.js

```diff
@@ -795,8 +809,8 @@ class Shard extends EventEmitter {
                     /**
                     * Fired when a guild member's voice state changes
                     * @event Client#voiceStateUpdate
-                    * @prop {Member} member The member
-                    * @prop {Object} oldState The old voice state
+                    * @prop {Member | Object} member The member. If the member is not cached and Discord doesn't send a member payload, this will be an object with `id` and `voiceState` keys. No other property is guaranteed
+                    * @prop {Object?} oldState The old voice state If the member. If the above caveat applies, this will be null
```

Comment:
Am I interpreting this right?

```suggestion
                    * @prop {Object?} oldState The old voice state of the member. If the above caveat applies, this will be null
```